src/main/kotlin/racingcar/domain/Race.kt

```diff
@@ -0,0 +1,25 @@
+package racingcar.domain
+
+class Race {
+    private val cars: List<Car>
+
+    constructor(carCount: Int) {
+        this.cars = (0 until carCount).map { Car() }
```

Comment:
리스트 크기와 인덱스를 기반으로 요소 값을 정의하는 초기화 함수를 사용하는 가짜 생성자(fake constructor)가 있습니다.
https://kotlinlang.org/docs/constructing-collections.html#initializer-functions-for-lists

```suggestion
        this.cars = List(carCount) { Car() }
```

Reply:
오! 완전 몰랐던 문법이네요 :) 감사합니다!